zipkin/src/main/java/zipkin/TraceKeys.java

```diff
@@ -99,6 +99,12 @@
    */
   public static final String HTTP_RESPONSE_SIZE = "http.response.size";
 
+  /**
+   * The query executed for JDBC call.  Ex. "select * from customers where id = ?"
```

Comment:
@jcarres-mdsol @basvanbeek @eirslett what say you? (asking for your non-java opinion)